<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RazorSysStatConfiguration</class>
 <widget class="QDialog" name="RazorSysStatConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Razor SysStat Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="graphGB">
       <property name="title">
        <string>Graph</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3" columnstretch="2,3">
        <property name="margin">
         <number>4</number>
        </property>
        <property name="spacing">
         <number>4</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="sizeL">
          <property name="text">
           <string>&amp;Minimal size</string>
          </property>
          <property name="buddy">
           <cstring>sizeSB</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="intervalL">
          <property name="text">
           <string>Update &amp;interval</string>
          </property>
          <property name="buddy">
           <cstring>intervalSB</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="titleL">
          <property name="text">
           <string>&amp;Title</string>
          </property>
          <property name="buddy">
           <cstring>titleLE</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="linesL">
          <property name="text">
           <string>&amp;Grid lines</string>
          </property>
          <property name="buddy">
           <cstring>linesSB</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="titleLE"/>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="linesSB"/>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="sizeSB">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Minimal width if the panel is horizontal.&lt;/p&gt;&lt;p&gt;Minimal height is the panel is vertical.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="suffix">
           <string> px</string>
          </property>
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>120</number>
          </property>
          <property name="value">
           <number>30</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="intervalSB">
          <property name="suffix">
           <string> s</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>0.500000000000000</double>
          </property>
          <property name="maximum">
           <double>60.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="dataGB">
       <property name="title">
        <string>Data</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4" columnstretch="2,3">
        <property name="margin">
         <number>4</number>
        </property>
        <property name="spacing">
         <number>4</number>
        </property>
        <item row="2" column="0" colspan="2">
         <widget class="QStackedWidget" name="dataSW">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="cpuP">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QCheckBox" name="useFrequencyCB">
              <property name="text">
               <string>Use &amp;frequency</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="memP">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="netP">
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <layout class="QGridLayout" name="gridLayout" columnstretch="2,3">
              <property name="spacing">
               <number>4</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="maximumL">
                <property name="text">
                 <string>Ma&amp;ximum</string>
                </property>
                <property name="buddy">
                 <cstring>maximumHS</cstring>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="2,3">
                <property name="spacing">
                 <number>4</number>
                </property>
                <item>
                 <widget class="QLabel" name="maximumValueL">
                  <property name="text">
                   <string>XXX KBs</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="maximumHS">
                  <property name="maximum">
                   <number>39</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="logarithmicCB">
                <property name="text">
                 <string>Lo&amp;garithmic scale</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="logScaleSB">
                <property name="suffix">
                 <string> steps</string>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>64</number>
                </property>
                <property name="value">
                 <number>4</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_6">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="typeCOB">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="maxVisibleItems">
           <number>3</number>
          </property>
          <item>
           <property name="text">
            <string>CPU</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Memory</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Network</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="sourceL">
          <property name="text">
           <string>&amp;Source</string>
          </property>
          <property name="buddy">
           <cstring>sourceCOB</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="typeL">
          <property name="text">
           <string>T&amp;ype</string>
          </property>
          <property name="buddy">
           <cstring>typeCOB</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="sourceCOB"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="coloursGB">
       <property name="title">
        <string>Colours</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_8" columnstretch="2,3">
        <property name="horizontalSpacing">
         <number>4</number>
        </property>
        <property name="verticalSpacing">
         <number>5</number>
        </property>
        <property name="margin">
         <number>4</number>
        </property>
        <item row="0" column="0">
         <widget class="QRadioButton" name="useThemeColoursRB">
          <property name="text">
           <string>Use t&amp;heme colours</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="useCustomColoursRB">
          <property name="text">
           <string>Use c&amp;ustom colours</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="customColoursB">
          <property name="text">
           <string>Custom colour ...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>intervalSB</tabstop>
  <tabstop>sizeSB</tabstop>
  <tabstop>linesSB</tabstop>
  <tabstop>titleLE</tabstop>
  <tabstop>typeCOB</tabstop>
  <tabstop>sourceCOB</tabstop>
  <tabstop>useFrequencyCB</tabstop>
  <tabstop>maximumHS</tabstop>
  <tabstop>logarithmicCB</tabstop>
  <tabstop>logScaleSB</tabstop>
  <tabstop>useThemeColoursRB</tabstop>
  <tabstop>useCustomColoursRB</tabstop>
  <tabstop>customColoursB</tabstop>
  <tabstop>buttons</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>accepted()</signal>
   <receiver>RazorSysStatConfiguration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>RazorSysStatConfiguration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>typeCOB</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>dataSW</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RazorSysStatConfiguration</sender>
   <signal>maximumNetSpeedChanged(QString)</signal>
   <receiver>maximumValueL</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>704</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>intervalSB</sender>
   <signal>valueChanged(double)</signal>
   <receiver>RazorSysStatConfiguration</receiver>
   <slot>saveSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizeSB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RazorSysStatConfiguration</receiver>
   <slot>saveSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>linesSB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RazorSysStatConfiguration</receiver>
   <slot>saveSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>titleLE</sender>
   <signal>editingFinished()</signal>
   <receiver>RazorSysStatConfiguration</receiver>
   <slot>saveSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useFrequencyCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>RazorSysStatConfiguration</receiver>
   <slot>saveSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logarithmicCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>RazorSysStatConfiguration</receiver>
   <slot>saveSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sourceCOB</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>RazorSysStatConfiguration</receiver>
   <slot>saveSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>704</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>typeCOB</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>RazorSysStatConfiguration</receiver>
   <slot>saveSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>704</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useCustomColoursRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>customColoursB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useThemeColoursRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>RazorSysStatConfiguration</receiver>
   <slot>saveSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>maximumNetSpeedChanged(QString)</signal>
  <slot>on_typeCOB_currentIndexChanged(int)</slot>
  <slot>on_sourceCOB_currentIndexChanged(int)</slot>
  <slot>on_maximumHS_valueChanged(int)</slot>
  <slot>saveSettings()</slot>
  <slot>on_customColoursB_clicked()</slot>
 </slots>
</ui>
